packages/docregistry/src/components.tsx

```diff
@@ -23,7 +23,7 @@ export function createReadonlyLabel(
       <span
         className="jp-ToolbarLabelComponent"
         title={trans.__(
-          `Document is permissioned read-only; "save" is disabled, use "save as..." instead`
+          `Document is read-only. "Save" is disabled; use "Save as…" instead`
```

Comment:
Why use semicolons? Wouldn’t the punctuation be neater? 
```
`Document is read-only. "Save" is disabled. Use "Save as…" instead`
```

Reply:
I'm not sold that this is necessarily neater. The semicolon links two closely related statements, so I think it is still appropriate to keep it in this notification.